### ISSUE TITLE
FR: MetaModels Item Content Element

### ISSUE THREAD

**Issue by tim-bec** [MetaModels/core#870]
Ich wünsche mir ein Content-Element (wie damals, im Catalog) mit dem einzelne MetaModels Einträge Redakteursfreundlich in einen Artikel eingebunden werden können (Beispiel: Ansprechpartner).

**Comment by m-knorr** [comment]
+1

**Comment by asaage** [comment]
:+1:

**Comment by Aybee** [comment]
Mmh, noch nie gebraucht.

**Comment by discordier** [comment]
Ist bereits beauftragt und kommt demnaechst.

**Comment by tim-bec** [comment]
\o/

**Comment by xantippe** [comment]
Hallo Tim, ggf. stehe ich auf´m Schlauch aber das müsste doch schon funktionieren - z.B. habe ich für die Terminliste nur von "Jean-Pierre" (ganz unten: http://www.scrum-events.de/referenten-und-coaches/articles/jean-pierre-berchez.html) ein Inhaltselement mit Auswahl des Namens für einen Filter.

In Deinem Fall würde statt der Terminliste nur ein Item ausgegeben werden.

**Comment by MacKP** [comment]
@xantippe ja, an sich geht das schon. Ist aber nicht so freundlich für Redakteure wie es sinnvoll wäre ;-) Es geht halt darum eine so einfache Möglichkeit vorzugeben, wie es früher beim Catalog mal gegeben hat. Ein Admin gibt alles vor und der Redakteur brauch nur noch anklicken, welches Modul, welches Item daraus und welche Darstellung. Nicht noch die ganzen Grundeinstellungen wie beim Liste-Modul.

**Comment by xantippe** [comment]
ahh... o.k.!
"Das Bessere ist der Feind des Gutem" ;-)

**Comment by fabian-ibr** [comment]
+1

**Comment by asaage** [comment]
gibts diesbezüglich was neues?
Ein ce_metamodel_item wäre mir gerade sehr willkommen...

**Comment by zonky2** [comment]
> Ist bereits beauftragt und kommt demnaechst.

es lässt sich leider nicht mehr raus finden, wer die Sache "sponsoren" wollte :-(

**Comment by zonky2** [comment]
**Fundraising möglich!**
https://now.metamodel.me/de/unterstuetzer/fundraising#redakteurmodul

**Comment by zonky2** [comment]
Das lässt sich als RST-CE oder Contao-Inhaltselement recht einfach und individuell bauen https://metamodels.readthedocs.io/de/latest/cookbook/specials/ce_element_for_editors.html
